eng/Subsets.props

```diff
@@ -63,7 +63,7 @@
     <DefaultLibrariesSubsets Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)' or
                                         '$(BuildTargetFramework)' == '' or
                                         '$(BuildAllConfigurations)' == 'true'">libs.native+</DefaultLibrariesSubsets>
-    <DefaultLibrariesSubsets>$(DefaultLibrariesSubsets)libs.ref+libs.src+libs.packages</DefaultLibrariesSubsets>
+    <DefaultLibrariesSubsets>$(DefaultLibrariesSubsets)libs.ref+libs.src</DefaultLibrariesSubsets>
```

Comment:
what is the workflow to just build packages no? `build.cmd libs -pack`?

Reply:
`build.cmd libs.src -pack` is probably the quickest and easiest one after this change is in. But as before `build.cmd libs -allconfigurations` also builds the packages. Another alternative is invoking the source project directly: `dotnet pack src\libraries\src.proj`.

Reply:
Should we add an entry on the docs regarding this?

Reply:
We definitely want to have a dedicated doc page about packaging to also document the new switches that I introduced to i.e. add placeholder files for .NETFramework/Xamarin, framework assembly references, the docs switch, etc. I would like to focus on documentation in a follow-up PR though.

Reply:
Ok sounds good. Just FYI, we already have docs dedicated to packaging. https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/libraries-packaging.md, that's why I thought it might make sense to update on this PR.

Reply:
That document is actually pretty well written :) I will add some parts to it in a follow-up :)